components/infobox/wikis/rainbowsix/infobox_team_custom.lua

```diff
@@ -12,6 +12,15 @@ local Template = require('Module:Template')
 local Variables = require('Module:Variables')
 
 local Team = Lua.import('Module:Infobox/Team', {requireDevIfEnabled = true})
+local Achievements = Lua.import('Module:Infobox/Extension/Achievements', {requireDevIfEnabled = true})
+
+local ACHIEVEMENTS_BASE_CONDITIONS = {
+	'[[liquipediatiertype::!Showmatch]]',
+	'[[liquipediatiertype::!Qualifier]]',
+	'[[liquipediatiertype::!Charity]]',
+	'([[liquipediatier::1]] OR [[liquipediatier::2]])',
+	'[[placement::1]]',
+}
```

Comment:
What about teams that are not competing in S or A Tier events? Based on the current implemintation, they will have no achievements show, rather than have achievements show that are more relevant to their level of competition. 

If the intetion is to only show the most notable events (S & A tier events) and not to show results relevant to the team, then ive got no issue. Just something to think about

Reply:
personally i think we only should show tier 1 (depending on wiki decision also tier 2) wins here
for the lower tier achievements there is achievements table on the page